plugin.php

```diff
@@ -120,4 +120,90 @@ function ($fields, $block_type, $type_registry) {
 	}
 }
 
-WPGraphQLGutenbergACF::instance();
+
+/**
+ * Initialize the plugin
+ *
+ * @return WPGraphQLGutenbergACF|void
+ */
+function init() {
+
+	/**
+	 * If dependencies are missing, do not initialize the code
+	 */
+	if ( false === can_load_plugin() ) {
+		// Show the admin notice
+		add_action( 'admin_init', __NAMESPACE__ . '\show_admin_notice' );
+
+		// Bail
+		return;
+	}
+
+	/**
+	 * Return the instance of WPGraphQLGutenbergACF
+	 */
+	return WPGraphQLGutenbergACF::instance();
+}
+
+add_action( 'init', '\WPGraphQLGutenbergACF\init' );
+
+
+/**
+ * Show admin notice to admins if this plugin is active but dependencies are missing
+ * are not active
+ *
+ * @return bool
+ */
+function show_admin_notice() {
+
+	/**
+	 * For users with lower capabilities, don't show the notice
+	 */
+	if ( ! current_user_can( 'manage_options' ) ) {
+		return false;
+	}
+
+	add_action(
+		'admin_notices',
+		function() {
+			?>
+			<div class="error notice">
+				<p>WPGraphQL, Advanced Custom Fields, WPGraphQL for Advanced Custom Fields and  WPGraphQL Gutenberg must be active for "wp-graphql-gutenberg-acf" to work</p>
```

Comment:
could you please wrap this as translation string?

Reply:
I added the code for the translation